sections/errorhandling/asyncerrorhandling.md

```diff
@@ -32,9 +52,33 @@ async function executeAsyncTask () {
   }
 }
 ```
+</details>
+
+<details>
+<summary><strong>Typescript</strong></summary>
+
+```typescript
+async function executeAsyncTask () {

```

Comment:
I didn't type the variables because the TS compiler should be able to infer the return types of the functions. However I'll add a return type to `executeAsyncTask`, good catch!